configs/virt/incus/scripts/buildimage.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+set -ex
+
+IMAGE_NAME=skiffos/testing
+CONTAINER=skiff
+OUTPUT_PATH="$BUILDROOT_DIR/output"
+IMAGES_PATH="$BUILDROOT_DIR/images"
+WORKING_PATH="${BUILDROOT_DIR}/nspawn-run"
+PERSIST_PATH="${WORKING_PATH}/nspawn-persist"
+roottar="${IMAGES_PATH}/rootfs.tar"
+
+command -v incus > /dev/null 2>&1 || {
+  echo "Failed to find the incus command. Is incus installed on your host system?"
+  exit 1
+}
+
+mkdir -p "${PERSIST_PATH}"
+TMPDIR="$(mktemp -d)"
+trap 'rm -rf "$TMPDIR";' EXIT
+mkdir -p "${TMPDIR}/rootfs"
+tar -xf "$roottar" -C "$TMPDIR/rootfs"
+pushd "$TMPDIR"
+creationtime="$(date +%s)"
+default_metadata="architecture: x86_64
+creation_date: $creationtime
+properties:
+  description: SkiffOS
+  os: Linux x86
+  release: unknown"
+echo "${INCUS_METADATA:-"${default_metadata}"}" > "${TMPDIR}/metadata.yaml"
+tar caf skiff.tar.gz rootfs metadata.yaml
+if incus image show "${IMAGE_NAME}" >/dev/null 2>&1
+then
+  incus image delete "${IMAGE_NAME}"
+fi
+incus image import --alias "${IMAGE_NAME}" skiff.tar.gz
```

Comment:
I'm fine with this, but, if an incus image is conceptually similar to a docker image, do we import it right away or write it to skiffos-incus.tar.gz in images/ and let the user import it later? I'm ok with either but let me know what you think.

Reply:
We can either import it here or in the run script, but it's necessary to run the container.

In any case, it makes sense to keep the generated tar.gz file to let users use that directly